content/_index.de.md

```diff
@@ -1,49 +1,56 @@
 
-**Organic Maps** ist heutzutage eine der wenigen Anwendungen, die 100% der Funktionen ohne aktive Internetverbindung unterstützt. Installiere Organic Maps, lade Karten herunter, wirf deine SIM-Karte weg (dein Betreiber verfolgt dich übrigens ständig) und mache einen Wochenausflug mit einer einzigen Akkuladung, ohne dass ein Byte an das Netz gesendet wird.
+**Organic Maps** ist heutzutage eine der wenigen Apps, die 100% ihrer Funktionen ohne aktive Internetverbindung unterstützt. Installiere Organic Maps, lade Karten herunter, wirf deine SIM-Karte weg (dein Betreiber verfolgt dich übrigens ständig) und mache einen Wochenausflug mit einer einzigen Akkuladung, ohne dass ein Byte an das Netz gesendet wird.
 
-### Organic Maps herunterladen und installieren von [AppStore][appstore], [Google Play][googleplay], [FDroid][fdroid], [Huawei AppGallery][appgallery] {#install}
+### Organic Maps von [AppStore][appstore], [Google Play][googleplay], [F-Droid][fdroid], [Huawei AppGallery][appgallery] herunterladen und installieren {#install}
 
 {{ badges(lang='de') }}
 
 {{ screenshot(src='/images/screenshots/hiking.jpg', alt='Wandern') }}
+
 {{ screenshot(src='/images/screenshots/prague.jpg', alt='Prag') }}
+
 {{ screenshot(src='/images/screenshots/search.jpg', alt='Offline Suche') }}
-{{ screenshot(src='/images/screenshots/dark.jpg', alt='Navigation im dunklen Modus') }}
 
-## Funktionen
+{{ screenshot(src='/images/screenshots/dark.jpg', alt='Navigation im dunklen
+Modus') }}
 
-Organic Maps ist die ultimative Begleit-App für Reisende, Touristen, Wanderer und Radfahrer:
+## Funktionen
 
-- Detaillierte Offline-Karten mit Orten, die auf anderen Karten nicht zu finden sind, dank [OpenStreetMap](https://osm.org)
-- Radrouten, Wanderwege und Spazierwege
-- Höhenlinien, Höhenprofile, Gipfel und Steigungen
-- Schritt-für-Schritt-Navigation für Fußgänger, Radfahrer und Autos mit Sprachführung
-- Schnelle Offline-Suche auf der Karte
-- Export und Import von Lesezeichen im KML/KMZ-Format (GPX ist [geplant](https://github.com/organicmaps/organicmaps/issues/624))
-- Dunkler Modus zum Schutz der Augen
+Organic Maps ist die ultimative Begleit-App für Reisende, Touristen,
+Wanderer und Radfahrer:
+
+- Detaillierte Offline-Karten mit Orten, die es auf anderen Karten nicht
+  gibt, dank [OpenStreetMap][openstreetmap]
+- Rad-, Wander- und Spazierwege
+- Höhenlinien, Höhenprofile, Bergspitzen und Steigungen
+- Schritt-für-Schritt-Navigation für Fußgänger, Radfahrer und Autos mit
+  Sprachführung
+- Schnelle Offline-Suche von Orten auf der Karte
+- Export und Import von Lesezeichen im KML/KMZ-Format (GPX ist
+  [geplant][gpx_issue])
+- Dunkler Modus zum Schutz deiner Augen
 - Länder und Regionen nehmen nicht viel Speicherplatz in Anspruch
 - Kostenlos und Open-Source
 
-## Wieso Organic?
+## Warum Organic?
 
-Organic Maps ist rein und organisch, mit Liebe gemacht:
+Organic Maps ist rein und nachhaltig, mit Liebe gemacht:
```

Comment:
rein, nachhaltig und mit Liebe gemacht:

Reply:
Haven't we translated 'organic' with 'organisch' so far?

Reply:
Let's use Organic Maps in the title and Organisch in other places. It will help people to find the app.

Reply:
A bit off topic: I have not yet understood exactly how the glossary on weblate works, but perhaps we should add the term so that it is translated consistently depending on the language.